spamc.go

```diff
@@ -67,10 +68,148 @@ func (c *Client) send(
 	return conn, nil
 }
 
+// rearrangeMessageReader will rearrange the message reader to make sure SA will understand the message
+func (c Client) rearrangeMessageReader(message io.ReadSeeker, size int64) (io.Reader, int64, error) {
+	var headers bytes.Buffer
+	var rest bytes.Buffer
+
+	isASCII := func(m []byte) bool {
```

Comment:
Could you move this out of this function and add it as a "regular" function please?

Functionally it doesn't really matter, but this function is already rather large, and these eight lines don't really *need* to be here.

Reply:
Sure will do ;)